include/zephyr/bluetooth/audio/audio.h

```diff
@@ -173,6 +173,14 @@ enum bt_audio_metadata_type {
 	BT_AUDIO_METADATA_TYPE_VENDOR              = 0xFF,
 };
 
+/**
+ * Helper to check whether metadata type is known.
```

Comment:
```suggestion
 * Helper to check whether metadata type is known by the stack
```

I am unsure whether this is relevant to add as a public API. My comments were specifically about the case where the application may know more metadata types than the stack, in which case this macro won't do any good

Reply:
We already have macros like `BT_AUDIO_CONTEXT_TYPE_ANY` or `BT_AUDIO_LOCATION_ANY`.

Otherwise, the app has to implement `switch` with all the cases to reject the RFU to pass the qualification tests.

Reply:
That is a fair point. The application can just omit using this macro is it uses types unknown by the stack.